### ISSUE TITLE
Passing array from attributes to config in frontend resource does not render proper block in haproxy.cfg

### ISSUE THREAD

**Issue by wktmeow** [nathwill/chef-haproxy-ng#26]
When I run a wrapper cookbook and try to pass config settings from node attributes as an array, the description and bind lines are missing from the frontend block.

If I hard-code the config lines then it works as expected
attributes
```ruby
default['haproxy']['rpm_source'] = "https://some.package.repo/haproxy-1.5.14-1.el6.x86_64.rpm"
default['haproxy']['proxies'] = %w( TCP stats my_frontend my_backend )
default['haproxy']['frontend_config'] = [ 'option clitcpka', 'timeout client 60m' ]
```
default.rb cookbook
```ruby
remote_file "#{Chef::Config[:file_cache_path]}/haproxy.rpm" do
	source node['haproxy']['rpm_source']
	action :create
end

rpm_package 'haproxy' do
	source "#{Chef::Config[:file_cache_path]}/haproxy.rpm"
	options "--nomd5"
end

haproxy_listen 'stats' do
	mode 'http'
	bind '*:1936'
	config [
	'stats enable',
    'stats hide-version',
    'stats realm Haproxy\\ Statistics',
    'stats uri /'
	]
end

haproxy_defaults 'TCP' do
  mode 'tcp'
  balance 'leastconn'
  source node['ipaddress']
  config [
    'option clitcpka',
    'option srvtcpka',
    'timeout connect 5s',
    'timeout client 300s',
    'timeout server 300s'
  ]
end

haproxy_backend "my_backend" do
    mode 'tcp'
    description 'backend server pool'
    balance 'roundrobin'
    source node['ipaddress']
    servers [
    {
      'name' => 'app01',
      'address' => '12.34.56.78',
      'port' => 80,
      'config' => 'check inter 5000 rise 2 fall 5'
    },
    {
      'name' => 'app02',
      'address' => '12.4.56.78',
      'port' => 80,
      'config' => 'check inter 5000 rise 2 fall 5'
    }
    ]

end

haproxy_frontend "my_frontend" do
  mode  'tcp'
  description 'frontend for some stuff'
  bind '*:80'
  default_backend 'my_backend'
  config node['haproxy']['frontend_config']
end

include_recipe "haproxy-ng"
```

The config block for the frontend reads as:
```
frontend my_frontend
  option clitcpka
  timeout client 60m
```

Strangely, the temporary file in the chef cache does look to have the correct values:
```
cat haproxy.frontend.my_frontend.cfg
frontend my_frontend
  bind *:80
  mode tcp
  option clitcpka
  timeout client 60m
  description frontend for some stuff
```

During the chef run, it looks like the template is removing the lines:
```
Recipe: haproxy-ng::default
  * haproxy_instance[haproxy] action createRecipe: <Dynamically Defined Resource>
  * template[haproxy-instance-haproxy] action create
    - update content in file /etc/haproxy/haproxy.cfg from 30d91b to b6d0dd
        --- /etc/haproxy/haproxy.cfg	2015-12-03 20:51:41.986550122 -0700
        +++ /tmp/chef-rendered-template20151203-19053-1a6zage	2015-12-03 20:52:00.492648487 -0700
        @@ -26,8 +26,6 @@
           stats uri /

         frontend my_frontend
        -  description some frontend
        -  bind *:80
           option clitcpka
           timeout client 60m
```

**Comment by nathwill** [comment]
@wktmeow thanks for the report! that's very strange indeed... i'll take a look and see what i can find.

**Comment by nathwill** [comment]
so i'm still looking into this, but for right now, you should be able to work around it by calling #to_a on the attribute, like:

```ruby
haproxy_frontend "my_frontend" do
  mode  'tcp'
  description 'frontend for some stuff'
  bind '*:80'
  default_backend 'my_backend'
  config node['haproxy']['frontend_config'].to_a
end
```

**Comment by nathwill** [comment]
this should be addressed by #27

**Comment by wktmeow** [comment]
Awesome, thanks so much for the workaround and the fix!
